persistence-modules/spring-boot-persistence-4/pom.xml

```diff
@@ -79,13 +67,24 @@
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
+                <configuration>
+                    <mainClass>com.baeldung.customfunc.CustomFunctionApplication</mainClass>
+                </configuration>
+            </plugin>
+            <plugin>
```

Comment:
Could you try to remove the explicit plugin configuration here?
I guess this is already passed from parent pom

Reply:
Changes done.